### ISSUE TITLE
if (typeof onScroll !== 'function')

### ISSUE THREAD

**Issue by SixYearsInFebruary** [cheton/react-infinite-tree#9]
I found the below code in InfiniteTree.jsx at 256 line, which is a bug @cheton .
```
if (typeof onScroll !== 'function') {  
  virtualListProps.onScroll = onScroll;  
}
````
or......you deliberately do that?
ps. do you have any plan to support the checkbox in infinite-tree? thx.
I have read the (creating-tree-nodes-with-checkboxes)[https://github.com/cheton/infinite-tree#creating-tree-nodes-with-checkboxes], but it could not support the relationship of checkbox status between the parent node and children nodes.

**Comment by cheton** [comment]
This is a regression bug in v1.0-alpha. I will fix it soon.

How about adding `tree.checkNode(node)` and `tree.uncheckNode(node)` to infinite-tree? Then you can call tree.checkNode(node) or tree.uncheckNode(node) to recursively update checkbox status and indeterminate state of parent/child nodes.

**Comment by cheton** [comment]
Fixed in v1.0.0-alpha.3
